### PR TITLE
Add aci_rest_managed resource and data source

### PR DESCRIPTION
**Pull request by lhercot**

### CODE REVIEW

File: aci/resource_aci_rest_managed.go [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+	content := d.Get("content")
+	contentStrMap := toStrMap(content.(map[string]interface{}))
+	newContent := make(map[string]interface{})
+	restContent, ok := c.Search("imdata", className, "attributes").Index(0).Data().(map[string]interface{})
+
+	if !ok {
+		return diag.Errorf("Failed to retrieve REST payload for class: %s.", className)
+	}
+
+	for attr, value := range restContent {
```
Comment (xinyuezhao) [inline aci/resource_aci_rest_managed.go:116]: what are these restContent and contentStrMap?

File: website/docs/r/rest_managed.markdown [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+layout: "aci"
+page_title: "ACI: aci_rest_managed"
+sidebar_current: "docs-aci-resource-rest-managed"
+subcategory: ""
+description: |-
+  Manages ACI Model Objects via REST API calls. This resource can only manage a single API object and its direct children. It is able to read the state and therefore reconcile configuration drift.
+---
+
+# aci_rest_managed (Resource)
+
```
Comment (xinyuezhao) [inline website/docs/r/rest_managed.markdown:12]: do we want to add some info to tell user in which case use this resource while in other cases use aci_rest resource?

File: aci/resource_aci_rest_managed.go (hunk 1 above)
Comment (danischm) [reply aci/resource_aci_rest_managed.go:116]: `contentStrMap` is a map of attributes retrieved from TF state. `restContent` is a map of attributes as returned from the REST GET API call. Both of them are then used to compile an updated map of attributes (`newContent`).

**Review by shrsr — APPROVED** [review]

File: website/docs/r/rest_managed.markdown (hunk 2 above)
Comment (lhercot) [reply website/docs/r/rest_managed.markdown:12]: We will recommend the use of aci_rest_managed whenever people ask but I don't think we should add anything to the documentation of the resource itself.

**Review by xinyuezhao — APPROVED** [review]

**Pull request merged**